lib/snap-fingers.js

```diff
@@ -25,6 +30,24 @@ module.exports = () => {
 		logger.log(`😈 Snapping fingers inside '${cwd}' with full power...`);
 	}
 
+	const ironManWithGauntlet = readArgs(constants.arguments.ironManWithGauntlet);
+
+	if(ironManWithGauntlet) {
+		logger.log(`I... am... Iron Man!`);
+		const correctPaths = JSON.parse(fs.readFileSync(`${os.homedir()}/.to-dust/correct-paths.json`, 'utf8'));
+
+		correctPaths.forEach((filePath) => {
+			let splitFilePath = filePath.file.split('/');
+			let filename = splitFilename.join('-');
+			fs.rename(aFile.file, `${os.homedir()}/.to-dust/${filename}`, (err) => {
```

Comment:
Have you tested this? `aFile` is undefiend here. :/

Reply:
Also, the sequence of argument would be reverse. We would want to move files from `.to-dust` to its original location

Reply:
Sorry, actually I was having a test branch where I was doing all this and after the fork, I just copied pasted stuff around which caused these issues.

Reply:
Let me just go through the whole thing comparing with my test folder and I'll open the PR again :/

Reply:
I've tested it and it works now, do let me know if you find any more issues :)